pkg/common/args.go

```diff
@@ -48,6 +48,12 @@ func PrintInfo(clientSet *kubernetes.Clientset, config *rest.Config) {
 // ValidateArgs validates the arguments passed to the program
```

Comment:
above comment is for ValidateArgs() method. I think you have not added it in right place

Reply:
Oh I missed it... Let me take a look at it..

Will push the changes soon..